test/integration/dockerregistryclient_test.go

```diff
@@ -224,14 +222,19 @@ func doTestRegistryClientImage(t *testing.T, registry, version string) {
 	}
 }
 
+func TestRegistryClientImageV2ManifestV2(t *testing.T) {
+	t.Skip("openshift/schema-v2-test-repo is being pushed by Docker 1.10, which causes this test to fail because digests don't match")
```

Comment:
@soltysh I believe that to check the docker version and skip the test in this case is a hack. Nevermind. After [commit](https://github.com/openshift/origin/pull/10003/commits/c452be1ac0d1278c9dc236059db55c903d642bae) it's no longer needed anymore. All tests must work without exceptions.

Moreover, before this fix, our client has always worked using v1 schema.